chromatic/rainbows/actions/shift.py

```diff
@@ -1,23 +1,33 @@
 from ...imports import *
 
-__all__ = ['shift']
+__all__ = ["shift"]
 
-def shift(self, velocity=5*u.km/u.s):
+
+def shift(self, velocity=0 * u.km / u.s):
     """
-    Apply a relativistic longitudinal doppler effect to the wavelength array
+    Doppler shift the wavelengths of this Rainbow.
+    Positive velocities make wavelengths longer (redshift).
+    Negative velocities make wavelengths shorter (bluesfhit).
 
     Parameters
     ----------
-    velocity
-        the systemic velocity in question, in km/s
+    velocity : astropy.units.Quantity
```

Comment:
The nice thing about astropy units is that the quantity doesn't have to be in `km/s`. It just needs to have units that can convert to `km/s`!